uwblueprint/paramedics-react
# CCP Dashboard Audit

[Notion ticket](https://www.notion.so/uwblueprintexecs/Audit-CCP-Dashboard-83dd93a731684a0f8e5104d7b2fb4e4b)
[Associated Backend PR](https://github.com/uwblueprint/paramedics-web/pull/88)

## Changes
- Refer to Notion ticket

## Screenshots
iPad Pro
![image](https://user-images.githubusercontent.com/41495076/104821862-d1f0c080-580c-11eb-9045-8b526b231696.png)

iPad
![image](https://user-images.githubusercontent.com/41495076/104821873-e7fe8100-580c-11eb-98fc-a598f43ecb1d.png)

Aligned triage field
![image](https://user-images.githubusercontent.com/41495076/104821951-7b37b680-580d-11eb-984c-03e3544e170c.png)

Patient dialog
![image](https://user-images.githubusercontent.com/41495076/104821964-8b4f9600-580d-11eb-8e91-aa7aae6992b2.png)

## Testing
- Ensure CCP dashboard elements are properly sized in iPad/iPad Pro (landscape orientation)
- Ensure patients table includes CTAs field and Last Edited field puts appropriate time (as explained in Notion ticket)
- Ensure the triage field in the patient's table for a deleted patient is aligned with un-deleted patients
- Ensure patient dialog:
    - Includes CTAS, Ambulance, Last Edited fields
    - All triage colors are now in black
    - Aligned labels and values (as shown in Notion ticket)
- Ensure patient CTA can be edited after clicking "Edit" in the patient dialog

## Checklist
>[Code review doc for reference](https://www.notion.so/uwblueprintexecs/Code-Review-a21fc85b00394f488e92d9d605f6b2bc)

before opening PR

filling out PR

after opening PR
